Sources/MapboxCoreNavigation/EHorizon/RoadGraphPath.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import CoreLocation
+import MapboxNavigationNative
+
+extension RoadGraph {
+
+    /** A position along a linear object in the road graph. */
+    public struct Path {
+        /** The edge identifiers that form the linear object. */
+        public let edgeIdentifiers: [ElectronicHorizon.Edge.Identifier]
+
+        /** The distance from the start of the linear object to the user’s location as a fraction of the linear object’s length from 0 to 1. */
+        public let fractionFromStart: Double
+
+        /** The distance from the the user’s location to the end of the linear object as a fraction of the linear object’s length from 0 to 1. */
+        public let fractionToEnd: Double
```

Comment:
Isn't it equal `1 - fractionFromStart`? Do we need both values?

Reply:
Maybe not, and also it would be more intuitive to provide a distance from the start. @SiarheiFedartsou, can we assume that the `length` is always consistent with the percentages? In other words, is there any reason the length might not cover the whole path but the percentages would?

Reply:
Not exactly. I made an image with some explanation what actually "graph path" mean:
<img width="830" alt="Screen Shot 2021-03-16 at 10 08 57" src="https://user-images.githubusercontent.com/266271/111270435-e7357000-8640-11eb-9a14-c728c05a592a.png">

Reply:
Also I'd change docstrings for `RoadGraphPath`. It is actually abstract type and it is not necessary [to be connected with current user position](https://github.com/mapbox/mapbox-navigation-ios/blob/78b5de684c512d23065a161251b3f187b1a7597a/Sources/MapboxCoreNavigation/EHorizon/RoadGraphPath.swift#L15).
See our docs https://github.com/mapbox/mapbox-navigation-native/blob/ad622b030121846f55deb1f05edbdd58166ab8e6/bindings/common-types.mbg#L109

Reply:
Fixed in 327b397fa9831c321f37c11fa17b580f1a2cbc90.